content/en/docs/concepts/storage/volumes.md

```diff
@@ -1030,6 +1030,57 @@ spec:
       fsType: ext4
 ```
 
+#### vSphere raw block volume support
+Starting with release 1.13, the vSphere storage plugin includes raw block volume support.  The following shows an example configuration.
```

Comment:
Is there anything different here that is specific to vsphere?  If not, can we just add vsphere to the list of plugins that support block?

Also, is it alpha or beta?

Reply:
@msau42 Yes there are three configurations in setup that is vsphere specific in the yamls (`StorageClass:provisioner`, `PVC:annotation`, and `PV:vsphereVolume`).

This would be `alpha` for in-tree block support.

Reply:
I mean, are those any different from filesystem vsphere volume support?

Reply:
Can you also include this tag to denote it's alpha?
```
{{< feature-state for_k8s_version="v1.13" state="alpha" >}}
```

Reply:
@msau42 ok I see what you mean, setup is no different from filesystem.  I fixed block support to be inline with other in-tree driver support as suggested.